vmcnet/updates/proxsr.py

```diff
@@ -73,12 +73,12 @@ def proxsr_update_fn(
 
 def constrain_norm(
     grad: P,
-    learning_rate: chex.Numeric,
     norm_constraint: chex.Numeric = 0.001,
+    learning_rate_correction: chex.Numeric = 1.0,
```

Comment:
Param seems a bit oddly named given it doesn't really have to do with the learning rate any more. Could just remove it for now since it looks unused?